lib/msf/ui/console/command_dispatcher/core.rb

```diff
@@ -1395,6 +1413,34 @@ def cmd_sessions(*args)
           sid = nil
         end
       end
+
+   #launch server that will host meterpreter Web interface
+    when 'web_ui'
+      session = verify_session(sid)
+      if session
+        if session.respond_to?(:response_timeout)
+          last_known_timeout = session.response_timeout
+          session.response_timeout = response_timeout
+        end
+        print_status("Starting interaction with #{session.name}...\n") unless quiet
+        begin
+
+          Thread.new{
+            print_line "Starting Sinatra Server"
+            Msf::Server.run!
+          }.join
+          print_line("Opening WebConsole on #{session.type} session #{session.sid} (#{session.session_host})")
+          Rex::Compat.open_webrtc_browser('127.0.0.1:3000')
+        ensure
+          if session.respond_to?(:response_timeout) && last_known_timeout
+            session.response_timeout = last_known_timeout
+          end
+        end
+      else
+        sid = nil
+      end
+    end
```

Comment:
One too many `end` statements, ruby `when` blocks do not expect an end, this closes the `case` statement causing failure of console launch at the moment.

Reply:
fixed this issue in the latest Commit.